### ISSUE TITLE
Don't show membership changes as unread messages

### ISSUE THREAD

**Issue by LinAGKar** [vector-im/element-android#1941]
Currently, when there is a membership change in a room (e.g. someone joins or leaves, or a member changes display name), that shows up as an unread message, so the room gets the green unread bar, and the membership change is shown as the last message in the room tab. This causes a lot of noise, as many rooms are shown as unread even though there are no new messages. Instead only actual messages should cause a room to be shown as unread, and only actual messages should be shown as the most recent message.

**Comment by ralaud** [comment]
I think an option to disable these kind of notifications in Notification-Option instead of completely disabling them would be good.

**Comment by LinAGKar** [comment]
Yes, that would be great. I have nothing against still showing them in the timeline though.

**Comment by avalenn** [comment]
This seems a duplicate of https://github.com/vector-im/element-android/issues/1231
